src/Biscuit.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BiscuitApp;
+class Biscuit
+{
+
+}
```

Comment:
missing:
```php
require_once 'vendor/autoload.php';
```

also in the other two classes

Reply:
might be slightly different path

Reply:
resolved